crypto_secretstream/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "crypto_secretstream"
+version = "0.1.0"
+description = """
+Pure Rust implementation of libsodium's crypto_secretstream secret-key using
+chacha20 and poly1305
+"""
+authors = ["C4DT", "RustCrypto Developers"]
+edition = "2018"
+license = "Apache-2.0 OR MIT"
+readme = "README.md"
+documentation = "https://docs.rs/crypto_secretstream"
+homepage = "https://github.com/RustCrypto/nacl-compat"
+repository = "https://github.com/RustCrypto/nacl-compat/tree/master/crypto_secretstream"
+categories = ["cryptography"]
+keywords = ["nacl", "libsodium", "public-key", "chacha20", "poly1305"]
+
+[dependencies]
+aead = { version = "0.4", features = ["stream"] }
+rand_core = "0.6"
+chacha20 = { version = "0.7", features = ["expose-core", "hchacha"] }
+poly1305 = "0.7"
+snafu = "0.6"
```

Comment:
Is `snafu` really necessary?

Reply:
Not really, I moved to using a dedicated `errors` module.